### DISCUSSION TITLE
getting_started/install

### DISCUSSION THREAD

**Discussion by giscus** [py32duino/document#4] [comment]
# getting_started/install

此文档旨在介绍关于 PY32Duino 的相关安装方式。 Arduino IDE 的安装 Arduino IDE 支持 Windows、Mac OS X 及 Linux 等平台，下载地址为 Software > Arduino (https://www.arduino.cc/en/software)，本文仅介绍 Windows 下的安装方式，其他系统大...

https://arduino.py32.org/getting_started/install.html

**Comment by sunien** [comment]
在其它开发板管理器地址中输入 PY32Duino 的地址存在问题，是地址没有了吗？
Some indexes could not be updated. Server responded with:404 Not Found: https://arduino.py32.halfsweet.cn/package_py32_cn_index.json

**Reply by changlin1990** [reply]
同上，国内用不了吗？
